File: 2bPrecise/ClientApp/src/app/shared/models/employee.model.ts
export interface IEmpyloyee {
  id: number;
  firstName: string;
  lastName: string;
  position: number;
  imgUrl: string;
  managerId: number;
}

export class Employee {
  public id: number;
  public firstName: string;
  public lastName: string;
  public position: number;
  public imgUrl: string;
  public managerId: number;

  constructor(id: number, firstName: string, lastName: string, position: number, imgUrl: string, managerId: number) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.position = position;
    this.imgUrl = imgUrl;
    this.managerId = managerId;
  }

  getPositionText = () => {
    return this.position === 0 ? 'Employee' : 'Manager';
  }
}

File: 2bPrecise/Data/Models/EmployeeModel.cs
using _2bPrecise.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace _2bPrecise.Data.Models
{
    public class EmployeeModel
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public PositionType Position { get; set; }
        public string ImgUrl { get; set; }

        // Manager Info
        public int ManagerId { get; set; }
        //public string ManagerFirstName { get; set; }
        //public string ManagerLastName { get; set; }
        //public PositionType ManagerPosition { get; set; }
        //public string ManagerImgUrl { get; set; }
    }
}
